e2e/test/netstandard20/ProvisioningServiceClientE2ETests.cs

```diff
@@ -26,41 +22,39 @@ public enum AttestationType
         {
             Tpm,
             x509,
-            SymmetricKey
+            SymmetricKey,
```

Comment:
No, but with things like initializer lists, lists, and enums, it is good to append each line with a comma (allowed in C#) because one can add another line, re-order, and comment out other lines - all without changing the other lines.